### PR TITLE
Fix examples

### PR DESCRIPTION
**Pull request by Stefie**
Fixes https://github.com/polkadot-js/apps/issues/1061

### CODE REVIEW

File: packages/app-js/src/snippets/storage-examples.ts [hunk 1]
```diff
@@ -10,13 +10,16 @@ export const storageGetInfo: Snippet = {
...
-  api.query.timestamp.blockPeriod(),
+
+const [minimumPeriod, validators, transferFee] = await Promise.all([
+  api.query.timestamp.minimumPeriod(),
   api.query.session.validators(),
   api.query.balances.transferFee()
 ]);
 
-console.log('blockPeriod in seconds: ' + blockPeriod.toNumber());
+const period = util.formatNumber(minimumPeriod.toNumber() * 2);
```
Comment (Tbaut) [inline packages/app-js/src/snippets/storage-examples.ts]: (nit) Could we add a comment to explain the `* 2`, e.g it's related to aura implementation. This confused me at first :)

**Comment by jacogr** [comment]
The Aura stuff is problematic - we only get that info on a per block basis (where we have access to the engine type, or rather 4 byte magic) at some point the *2 is going to bite. (Not here, elsewhere)

File: packages/app-js/src/snippets/storage-examples.ts (hunk 1 above)
Comment (Stefie) [reply packages/app-js/src/snippets/storage-examples.ts]: Good point. My example is wrong either way... It should either be minimumPeriod in the console.log or block period ===  minimumPeriod*2 (with aura)

**Comment by Stefie** [comment]
Taking Jacos comment into account I changed the example to just getting the minimum period

**Pull request merged**